identity-service/src/notifications/index.js

```diff
@@ -1140,7 +1136,7 @@ class NotificationProcessor {
 
         // Based on this difference, schedule email for users
         // In prod, this difference must be <1 hour or between midnight - 1am
-        let maxHourDifference = 2 // 1.5
+        let maxHourDifference = 20 // 1.5
```

Comment:
@hareeshnagaraj is this correct?

Reply:
no this MUST be 1.5 or 2 before we check in